Extensions/Manage/Dnn.PersonaBar.Users/Components/UsersController.cs

```diff
@@ -72,7 +72,7 @@ public IEnumerable<UserBasicDto> GetUsers(GetUsersContract usersContract, bool i
         public IEnumerable<KeyValuePair<string, int>> GetUserFilters(bool isSuperUser = false)
         {
             var userFilters = new List<KeyValuePair<string, int>>();
-            for (var i = 0; i < 6; i++)
+            for (var i = 0; i < 9; i++)
```

Comment:
I know it was this way before, but in the interest of not breaking stuff in the future, can we get rid of the magic numbers here and have this switched to a foreach loop.  We are asking for trouble here with a hard-coded maximum value that is Max Enum value +1.

I know not your problem Peter, but trying to improve the quality as we go.  Sorry, I missed this on first review.

Reply:
I can rework that. Give me a few mins.

Reply:
BTW I agree with you and thought the same but I felt "touch as little as possible"

Reply:
Check new code ...